js/binance.js

```diff
@@ -2912,6 +2912,7 @@ module.exports = class binance extends Exchange {
          * @param {float} amount how much of currency you want to trade in units of base currency
          * @param {float|undefined} price the price at which the order is to be fullfilled, in units of the quote currency, ignored in market orders
          * @param {object} params extra parameters specific to the binance api endpoint
+         * @param {str|undefined} params.marginMode 'cross' or 'isolated', for spot margin trading
```

Comment:
Can you please add the new docstring types? 
Basically, `int` and `float` remain as is, `str` is now `string` and `dict` should be listed as `object`